### DISCUSSION TITLE
Logo of SocialRepo

### DISCUSSION THREAD

**Discussion by Param302** [Param302/SocialRepo#4] [Ideas]
## Description
I am thinking of a minimal logo, which should match the color scheme.
I don't have much knowledge about designs, so open to all suggestions. 😅

## Idea
I am having 2 ideas in my mind:
1. A simple neat logo just having the name Social Repo.
2. An inbox having stack of multiple social media icons in a translated manner.

If you have any other idea, you're welcome to share.

_Let's design the logo of SocialRepo_ 🙌

**Comment by porwalshreyaa** [comment]
Hey, Let's define the color scheme first.

**Reply by Param302** [reply]
Yes, btw I don't have much knowledge about designing.
But I have created 2 designs yesterday. Please have a look on [them](https://github.com/Param302/SocialRepo/discussions/4#discussioncomment-9327051).

**Reply by Param302** [reply]
Hey @porwalshreyaa ,
I tried this color scheme and it's looking good imo.
```
purple: #7029FF
white:  #FFFFFF
```
The background will be purple and name will be white, no border, just square a little rounded shape.
What are your thoughts?

**Reply by Param302** [reply]
### Sample
![image](https://github.com/Param302/SocialRepo/assets/76559816/37c7e945-2acd-4356-bc58-35af12578716)

**Comment by Param302** [comment]
These 2 designs I have worked on, but I am thinking of some other design.

![SR](https://github.com/Param302/SocialRepo/assets/76559816/8a23d752-a68a-4a0a-919c-e8546fa7572c)
![S](https://github.com/Param302/SocialRepo/assets/76559816/758d4d55-e12d-4323-b65a-b3d41a34f984)

Here's the canva link:
https://www.canva.com/design/DAGEZDlW-u4/jaaEkBYEKP76I3t2Pmo3CA/edit?utm_content=DAGEZDlW-u4&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

**Reply by porwalshreyaa** [reply]
looks good to me, ig highligting the letters in the centre would do, but I think let's find some color scheme from internet first like this... [these are some sites + their color scheme, should pick something out of these](https://visme.co/blog/website-color-schemes/)

**Reply by Param302** [reply]
I am thinking of some another design. 😅
For colors, try finding some cool color palette having blue or purple.

**Reply by porwalshreyaa** [reply]
I see that kind of purple, blue, dark pallate, got it! 😅

**Reply by Param302** [reply]
No, not dark, light only either blue or purple, with white combination.
Max of 3 colors.

**Reply by porwalshreyaa** [reply]
[This one has 5, you can pick 3 out of them](https://pin.it/2uqxwrdNV)

**Reply by Param302** [reply]
any other palette you have? 😅

**Reply by porwalshreyaa** [reply]
[this?](https://in.pinterest.com/pin/4292562135136521/)

**Reply by porwalshreyaa** [reply]
[or this](https://in.pinterest.com/pin/613334043020147750/) and [another](https://in.pinterest.com/pin/422281208854786/)

**Reply by porwalshreyaa** [reply]
[ye pura page dekhlo](https://in.pinterest.com/pin/422281208854786/)

**Reply by Param302** [reply]
these are a little dull, like not able to highlight..
Let me share the color palette, I think it will be good.

**Comment by porwalshreyaa** [comment]
> try finding some cool color palette having blue or purple.

[see this](https://muffingroup.com/blog/purple-color-palette/)

**Reply by Param302** [reply]
No 😅
Too many colors...
